rskj-core/src/main/java/co/rsk/core/RskFactory.java

```diff
@@ -259,6 +262,11 @@ public PersonalModule getPersonalModuleWallet(RskSystemProperties config, Rsk rs
         return new PersonalModuleWalletEnabled(config, rsk, wallet, pendingState);
     }
 
+    @Bean
+    public TxPoolModule getTxPoolModule(RskSystemProperties config, Rsk rsk, PendingState pendingState) {
```

Comment:
Don't inject config. Also consider using an `@Component` annotation when a `@Bean` is not necessary.